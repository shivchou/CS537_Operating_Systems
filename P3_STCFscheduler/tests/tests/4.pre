./edit-makefile.sh ../solution/Makefile test_4 > ../solution/Makefile.test
cp -f tests/test_4.c ../solution/test_4.c
cd ../solution/
make clean
make -f Makefile.test clean
make -f Makefile.test xv6.img
make -f Makefile.test fs.img
cd ../tests
