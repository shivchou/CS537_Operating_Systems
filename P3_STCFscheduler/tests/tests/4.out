XV6_TEST: scheduler : STCF
XV6_TEST: parent start
XV6_TEST: proc 5 start
XV6_TEST: proc 5 end
XV6_TEST: proc 4 start
XV6_TEST: proc 4 end
XV6_TEST: proc 3 start
XV6_TEST: proc 3 end
XV6_TEST: proc 0 start
XV6_TEST: proc 0 end
XV6_TEST: proc 1 start
XV6_TEST: proc 1 end
XV6_TEST: proc 9 start
XV6_TEST: proc 9 end
XV6_TEST: proc 6 start
XV6_TEST: proc 6 end
XV6_TEST: proc 2 start
XV6_TEST: proc 2 end
XV6_TEST: proc 7 start
XV6_TEST: proc 7 end
XV6_TEST: proc 8 start
XV6_TEST: proc 8 end
